source/Dgame/Graphic/Color.d

```diff
@@ -27,6 +27,144 @@ private:
 
 import derelict.sdl2.types;
 
+mixin template Colors(T) {
```

Comment:
Either a constraint that T has to be Color4b/Color4f or make the template private.

Reply:
Template is under the private label on line 26.